src/main/java/com/iemr/admin/data/employeemaster/V_Userservicerolemapping.java

```diff
@@ -147,9 +147,14 @@ public class V_Userservicerolemapping {
 	@Transient
 	private String[] villageName;   
 	
+	/*
+	 * @Expose
+	 * 
+	 * @Column(name="isSanjeevani") private Boolean isSanjeevani;
+	 */
     @Expose
-	@Column(name="isSanjeevani")
-	private Boolean isSanjeevani;
+	@Column(name="teleConsultation")
```

Comment:
Comment says isSanjeevani but the boolean is teleConsultation.
Also the attribute isSanjeevani is not needed?
If you are adding new attributes, can you also raise a PR for AMRIT-DB